student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] In the `initialize` method for the `Student` class, you only specified one `argument` to be taken. Whereas, the requirements state that you should specify 4 `arguments`: `age`, `name`, `classroom` and `parent_permission`. Also, in your `super` method in the `initialize` method, you need to specify that the `parent_permission` taken in should be the value for the `parent_permission` argument in the super method. 

- The problem with the way you have set the `Student` class up is that I would get an undefined variable error for `parent_permission` and even if I set the `parent_permission` variable to be `true` or `false`, the values for `name` and `age` will be equal to `nil` since I can't pass it through as a parameter when creating a new instance of the `Student` class. This will create an issue because, by nomenclature, every instance of the `Student` class should have these attributes.  

#### Hint: 
```ruby
def initialize(classroom, age, name = 'Unknown', parent_permission)
  super(age, name, parent_permission: parent_permission)
  @classroom = classroom
end
```

Reply:
I'll change that, thank you